tests/src/Functional/MessageEntityDelete.php

```diff
@@ -223,7 +229,7 @@ public function testReferencedEntitiesDelete() {
     $message->save();
 
     $account->delete();
-    $this->assertFalse(Message::load($message->id()), 'Message deleted after deleting single referenced user.');
+    $this->assertEquals(4, $delete_queue->numberOfItems(), 'Message queued for deletion after deleting single referenced user.');
   }
```

Comment:
At the end here, let's run the queue (cron should suffice), and verify they are indeed gone.